tests/test_cdk_builder.py

```diff
@@ -278,15 +278,20 @@ def fake_subprocess_run(self, command, *, env, shell, stdout, check, cwd):
         self.assertTrue(shell)
         self.assertTrue(check)
         self.assertIs(sys.stderr, stdout)
-        parser = argparse.ArgumentParser(prog="npx", exit_on_error=False)
```

Comment:
The `exit_on_error` was introduced in Python 3.9 whereas Sceptre still supports Python 3.8.

Reply:
Another possible fix could be of course to remove Python 3.8 from the test matrix here.